### DISCUSSION TITLE
Publishing dashboards as standalone HTML?

### DISCUSSION THREAD

**Discussion by railfan1975** [quarto-dev/quarto-cli#8384] [Q&A] [answered]
### Description

Will dashboards include an option to publish as standalone HTML files?

**Comment by mcanouil** [comment, marked as answer]
`embed-resources` does not work?

**Reply by railfan1975** [reply]
I will try this and see.  Thank you.

**Reply by railfan1975** [reply]
That does work!  Would it be possible to add this to the documentation on the next update?

**Reply by mcanouil** [reply]
Of course, the documentation will include comprehensive documentation of the new features.
